src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CalculatorTest {
+    private Calculator calculator;
+    
+    @Before
+    public void setCalculator(){
+        calculator = new Calculator();
+    }
+
+    @Test
+    public void sumTest() {
+        double result = calculator.calculete(20,10,'+');
```

Comment:
```suggestion
        double result = calculator.calculete(20, 10,' +');
```
після коми бажано додати пробіли скрізь)